### ISSUE TITLE
Indentation rule doesn't like alignment in multiline structures

### ISSUE THREAD

**Issue by Lexicality** [sasstools/sass-lint#591]
In structures such as
```scss
#loading-screen {
    // Magic animations
    visibility: hidden;
    transform: translateX(-100%);
    // Delay the visibility until we've drifted out
    transition: transform $in-out-speed,
                visibility 0s $in-out-speed;
    transition-timing-function: linear;
}
```
or
```scss
@font-face {
    font-family: "Freight Sans Pro";
    src: url("../fonts/FreightSansProMedium.woff") format("woff"),
         url("../fonts/FreightSansProMedium.ttf")  format("truetype");
}
```
scss-lint 1.5.1 complains about the overflow line's indentation:
```
error-pages/_fonts.scss
  4:10  error  Indentation of 9, expected 4  indentation
```
The only way to appease it is to reduce the overhanging line to the base indentation of the block
```scss
{
    src: url("../fonts/FreightSansProMedium.woff") format("woff"),
    url("../fonts/FreightSansProMedium.ttf")  format("truetype");
}
```
or put it all on one line
```scss
{
    transition: transform $in-out-speed, visibility 0s $in-out-speed;
}
```
neither of which I am happy with.

**Comment by bgriffith** [comment]
Thanks for flagging this up. We're aware of quite a few issues with the indentation rule and you can expect them to be addressed soon now that v1.6 is out of the door.

**Comment by DanPurdy** [comment]
Thanks @wbobeirne definitely a good idea to doc it here as I've just got it in my head at the minute!

**Comment by natorojr** [comment]
This may or may not be related to this issue, but I'm also seeing indentation issues with multiline nested structures:

```

**Comment by DanPurdy** [comment]
That's a separate issue @natorojr could you file a new issue please so we can keep track of it a little easier. Thanks.

**Comment by eolognt** [comment]
Fixing this would be very appreciated.

**Comment by pmccloghrylaing** [comment]
There's a few different ways people may want multiline indentation enforced.
```
// indent one level
{
    transition: transform $in-out-speed,
        visibility 0s $in-out-speed;
}
// perfect alignment
{
    transition: transform $in-out-speed,
                visibility 0s $in-out-speed;
}
// indent one level, enforce new line after ':'
{
    transition:
        transform $in-out-speed,
        visibility 0s $in-out-speed;
}
```
I've got a fork that enforces indenting one level: https://github.com/pmccloghrylaing/sass-lint/tree/indent-multiline-declaration, but it seems like there'll need to be an option for how multiline properties should be indented.

**Comment by Arial7** [comment]
I have exactly the same issue as @DaClan008 describes. This is very annoying....

**Comment by dominique-mueller** [comment]
Any updates on this? It's quiet annoying not being able to use multi-line indention ...

**Comment by nicklee** [comment]
We have the same issue here, would be great to get an update on when this will be implemented, single line multi transitions make me sad ;)

**Comment by DanPurdy** [comment]
@nicklee @dominique-mueller I haven't had any time to even look at this again recently, if someone wants to work on it and fix it then be my guest. Otherwise it unfortunately will have to wait until I get time again.

**Comment by jhpratt** [comment]
@DanPurdy Not a major issue for me, but now that `display: grid` is widely supported, multiline rules will be incredibly common.

**Comment by rudidude86** [comment]
Just to expand on @jhpratt's comment, it's beneficial to have well-aligned, multiline values for the [grid-template-areas](https://developer.mozilla.org/en-US/docs/Web/CSS/grid-template-areas) property in particular. The arrangement of the values is expressive -- you kind of end up with ASCII-art for how your template is laid out:

```css
grid-template-areas: "head head"
                     "nav  main"
                     "nav  foot";
```

As CSS Grid becomes more popular, I'm sure a lot of the community would find it beneficial to be able to lint for this situation!

Thanks!

**Comment by jhpratt** [comment]
That's exactly what I was referring to. Should've spelled it out ─ thanks for that.

**Comment by mysterycommand** [comment]
Any update on this? I see the epic (#680), but it appears locked and private.

**Comment by 64276** [comment]
Nothing yet? I'm still having the same issue.

**Comment by miquelferrerllompart** [comment]
Same here, some news?

**Comment by nwpappas** [comment]
This is a situation that I've been hoping to find a solution for. 

![image](https://user-images.githubusercontent.com/2529208/50987092-73760680-14bd-11e9-9d58-157078d458e4.png)

**Comment by beatrizsmerino** [comment]
I have the same issue:
I have temporarily disabled priority 2 but this is not really a fixed bug

![Captura de pantalla 2020-04-23 a las 12 14 19](https://user-images.githubusercontent.com/14045148/80087860-feb4c780-855b-11ea-913c-e52889446b2e.png)

![Captura de pantalla 2020-04-23 a las 12 17 25](https://user-images.githubusercontent.com/14045148/80088187-7256d480-855c-11ea-8415-77b5adc8cb07.png)

**Comment by lachieh** [comment]
Has anyone said it was fixed? I see a lot of people asking for an update but no one actually investigating a solution except a fork by @pmccloghrylaing from 4 years ago.

**Comment by Lexicality** [comment]
These days I let prettier handle the alignment rather than trying to deal with this rule.
